### PR TITLE
P002116-1286: Added documentation for dashboard introduction

### PR DESCRIPTION
**Pull request by suprishi**

### CODE REVIEW

File: documentation/home.asciidoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+== Overview
+
+The dashboard is your one stop destination for all your `devonfw-ide` needs. It serves as a UI on top of `devonfw-ide`.
+
+.dashboard
+image::images/collage.png["dashboard", width="600", link="images/collage.png"]
+
+Are you new to `devonfw`? You can download and setup the latest `devonfw-ide` from *dashboard* and link:home-page[get started]. Or if you prefer, you can also link:settings-page[download] one of our older `devonfw-ide` versions available from our maven repository. The dashboard will be a fantastic introduction to `devonfw` as it encapsulates all that `devonfw` offers.
+
+Or if you are already using `devonfw-ide`, you can link:settings-page[update] it's settings and software to the latest versions available.
```
Comment (sjimenez77) [inline documentation/home.asciidoc]: Found small typo: `it's settings` should be `its settings`

File: documentation/home.asciidoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+You also have the `devonfw` link:wiki-page[wiki] to know more about `devonfw` right from within the *dashboard*.
+
+== Motivation
+
+The `devon-ide` is a collections of tools and software which you can configure and customize as per your requirements. It ships with a number of command line tools under an umbrella `devon` command.
+
+The main motivation behind *dashboard* is to provide a user interface on top of this collection of command line tools. So it can do (almost) everything that the `devonfw-ide` could and more.
+
+It was also developed to serve as a single point of contact for the most common `devonfw` tasks. You can have multiple `devonfw-ide`s of different versions, each holding multiple projects and softwares configured differently. With *dashoard* you can manage all your projects from the same place.
```
Comment (sjimenez77) [inline documentation/home.asciidoc]: I think that you may have a small typo in the part you wrote "You can have multiple devonfw-ide's..." as it is not rendered right with asciidoctor.

**Review by sjimenez77 — CHANGES_REQUESTED** [review]
This introduction looks really well. Please @suprishi, check my comments before merging this PR. Thanks!

**Comment by suprishi** [comment]
Thank you @sjimenez77  for pointing out the typos. Corrected them and merging.

**Pull request merged**